Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/AnomalyDescriptor.java

```diff
@@ -40,9 +40,14 @@
     // information
     DiVector attribution;
 
+    long numberOfUpdates;
+
     // timestamp (basically a sequence index); kept as long for potential future use
     long timeStamp;
```

Comment:
Here and throughout the file, can we use `timestamp` instead of `timeStamp`?

Reply:
One word or two? changed to timestamp

Reply:
Thanks, "timestamp" should be one word I think.